rn/Teacher/ios/Teacher/AppDelegate+Routes.swift

```diff
@@ -125,49 +125,33 @@ extension AppDelegate {
             return PageDetailsViewController.create(context: ContextModel(.group, id: groupID), pageURL: pageURL, app: .teacher)
         })
 
-        let discussionDetails = { (props: Props) -> UIViewController? in
-            guard ExperimentalFeature.htmlDiscussions.isEnabled else {
-                return HelmViewController(
-                    moduleName: "/:context/:contextID/discussion_topics/:discussionID",
-                    props: props
-                )
+        if ExperimentalFeature.htmlDiscussions.isEnabled {
```

Comment:
I think this will break viewing a discussion module item. Could you test that out and make sure that still works? You can use `t` on `narmstrong`. CS 1400 > Modules.

Reply:
It seems to still work. Feature flag on or off

Reply:
Okay awesome!